### PR TITLE
Update versions of dependencies

### PR DESCRIPTION
**Pull request by dims**
Bump versions of dependencies we use

| Dependency | Notes | URL |
| ------------- | ------------- |------------- |
| github.com/containerd/ttrpc|v1.1.0| https://github.com/containerd/ttrpc/compare/v1.0.2...v1.1.0 |
| github.com/docker/docker|v20.10.18+incompatible| https://github.com/moby/moby/compare/v20.10.17...v20.10.18 |
| github.com/docker/go-units|v0.5.0| https://github.com/docker/go-units/compare/v0.4.0...v0.5.0 |
| github.com/karrick/godirwalk|v1.17.0| https://github.com/karrick/godirwalk/compare/v1.16.1...v1.17.0 |
| github.com/moby/sys/mountinfo| v0.6.2 | https://github.com/moby/sys/compare/mountinfo/v0.6.0...mountinfo/v0.6.2 | 
| github.com/moby/term|v0.0.0-20220808134915-39b0c02b01ae| https://github.com/moby/term/compare/3f7ff695adc6...39b0c02b01ae |
| github.com/opencontainers/runc|v1.1.4| https://github.com/opencontainers/runc/compare/v1.1.3...v1.1.4 | 
| github.com/stretchr/testify|v1.8.0| https://github.com/stretchr/testify/compare/v1.7.0...v1.8.0 |
| golang.org/x/net|v0.0.0-20210405180319-a5a99cb37ef4| Updated by go mod tidy |
| golang.org/x/sys|v0.0.0-20220520151302-bc2c85ada10a| Updated by go mod tidy |
| google.golang.org/grpc|v1.49.0| In sync with k8s |
| k8s.io/klog/v2|v2.80.1|In sync with k8s |

Signed-off-by: Davanum Srinivas <davanum@gmail.com>

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -7,33 +7,33 @@ require (
...
 	github.com/prometheus/common v0.10.0
-	github.com/stretchr/testify v1.7.0
-	golang.org/x/net v0.0.0-20210226172049-e18ecbb05110
-	golang.org/x/sys v0.0.0-20211116061358-0a5406a5449c
-	google.golang.org/grpc v1.33.2
-	k8s.io/klog/v2 v2.4.0
+	github.com/stretchr/testify v1.8.0
+	golang.org/x/net v0.0.0-20210405180319-a5a99cb37ef4
+	golang.org/x/sys v0.0.0-20220520151302-bc2c85ada10a
+	google.golang.org/grpc v1.47.0
```
Comment (bobbypage) [inline go.mod:36]: k8s uses `v1.49.0`, should cadvisor stay in sync?

https://github.com/kubernetes/kubernetes/blob/005a5ca08173b080193ecb01421e2513f32d045b/go.mod#L92

File: go.mod [hunk 2]
```diff
@@ -7,33 +7,33 @@ require (
 	github.com/Microsoft/go-winio v0.4.15
 	github.com/aws/aws-sdk-go v1.35.24
 	github.com/blang/semver v3.5.1+incompatible
-	github.com/containerd/ttrpc v1.0.2
+	github.com/containerd/ttrpc v1.1.0
 	github.com/containerd/typeurl v1.0.2
 	github.com/docker/distribution v2.8.1+incompatible // indirect
-	github.com/docker/docker v20.10.17+incompatible
+	github.com/docker/docker v20.10.18+incompatible
```
Comment (bobbypage) [inline go.mod:13]: k8s is on `v20.10.17`, do we plan to upgrade k8s to  `v20.10.18` as well?

https://github.com/kubernetes/kubernetes/blob/005a5ca08173b080193ecb01421e2513f32d045b/go.mod#L328

Comment (dims) [reply go.mod:13]: @bobbypage yep!

File: go.mod (hunk 1 above)
Comment (dims) [reply go.mod:36]: will fix this in cadvisor.

Comment (bobbypage) [reply go.mod:36]: Thanks, just to confirm will you update this PR with GRPC bump?

Comment (dims) [reply go.mod:36]: yep. in a few mins

Comment (dims) [reply go.mod]: Done!

**Comment by bobbypage** [comment]
LGTM, thanks for the updates!

**Review by bobbypage — APPROVED** [review]

**Pull request merged**